examples/prom/nonlinear_elasticity_global_rom.cpp

```diff
@@ -1363,13 +1489,13 @@ HyperelasticOperator::HyperelasticOperator(ParFiniteElementSpace& f,
     M->Assemble(skip_zero_entries);
     M->Finalize(skip_zero_entries);
     Mmat = M->ParallelAssemble();
-    HypreParMatrix* Me = Mmat->EliminateRowsCols(ess_tdof_list);
+    HypreParMatrix *Me = Mmat->EliminateRowsCols(ess_tdof_list);
     delete Me;
 
     M_solver.iterative_mode = false;
     M_solver.SetRelTol(rel_tol);
     M_solver.SetAbsTol(0.0);
-    M_solver.SetMaxIter(30);
+    M_solver.SetMaxIter(1000);
```

Comment:
Should it really take 1000 iterations?

Reply:
We had an issue where 30 iterations were too few so I just set it at a "high enough" value